src/test/scala/firrtlTests/interval/IntervalMathSpec.scala

```diff
@@ -19,24 +19,24 @@ class IntervalMathSpec extends FirrtlFlatSpec {
   val DShiftLeftPattern   = """.*output dshl.*<(\d+)>.*""".r
   val DShiftRightPattern  = """.*output dshr.*<(\d+)>.*""".r
   val ArithAssignPattern = """\s*(\w+) <= asSInt\(bits\((\w+)\((.*)\).*\)\)\s*""".r
-  def getBound(bound: String, value: Double): IsKnown = bound match {
```

Comment:
not related changes?

Reply:
`Range.Double` was deprecated in `2.12` and removed in `2.13`. Thus I changed it to `Range.BigDecimal` and it seemed silly  to convert from `BigDecimal` to `Double` and back to `BigDecimal`.